src/features/island/collectibles/components/DrCow.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { CollectibleProps } from "../Collectible";
+import { ITEM_DETAILS } from "features/game/types/images";
+
+export const DrCow: React.FC<CollectibleProps> = () => {
+  return (
+    <img
+      src={ITEM_DETAILS["Dr. Cow"].image}
+      style={{
+        width: `${PIXEL_SCALE * 19}px`,
```

Comment:
Generally we will make the width the same as the width of the file. so in this case the width of Dr. Cow is 25px.
You might need to update the positioning of the collectibles to make sure that they are centered
```suggestion
        width: `${PIXEL_SCALE * 25}px`,
```
![image](https://github.com/user-attachments/assets/8db08cbe-5419-44fc-b630-82e38adfd9e6)